Makefile

```diff
@@ -23,14 +29,38 @@ e2e: deploy
 $(TARGETS): vendor/modules.txt
 	./scripts/$@
 
-build-subctl: pkg/subctl/operator/common/embeddedyamls/yamls.go $(shell find pkg/subctl/ -name "*.go")
+clean:
+	rm -f $(BINARIES)
 
-bin/subctl: build-subctl
+build: operator-image binaries
```

Comment:
Yeah so I missed this before, but I think this is a major change.

Now everything relying on `build` would always run the cross compile..

In actuality it's not really needed for the e2e (just the one subctl of the current os/arch) but adds a few minutes to each job.

I'd rather we not have build-cross as default, but as a distinct target as it used to be, or make the e2e chain not rely on it all but just the subctl

Reply:
Fixed, it’s back to `build-cross`.